source/newcommitter.md

```diff
@@ -20,11 +20,46 @@ The [Contributor Ladder](https://community.apache.org/contributor-ladder.html) h
 
 {{% toc %}}
 
+<a name="NewCommitter-Summary"></a>
+### TL;DR - Inviting a new Committer
+
+1. Discuss the proposed committer (and optionally PMC member). (Use
+   [this template](templates/committer-discuss.txt).)
+If the discussion seems to be going positively,
+   call a vote. (Use [this template](templates/committer-vote.txt) for
+   committer, [this template](templates/committer-pmc-vote.txt) for
+   committer and PMC member.)
+1. Close the vote. (Use [this template](templates/committer-results.txt) for a committer vote, or [this template](templates/committer-pmc-results.txt) for a committer + PMC member vote.)
+1. If the result is positive, invite the new committer. (Use [this
+   template](templates/committer-invite.txt).)
+
+If they accept, then:
+
+1. If they already have an Apache id, grant appropriate commit privileges.
+   Use the Whimsy tool to update the roster via [Comitee Roster](https://whimsy.apache.org/roster/committee/) or
+   [PPMC Roster](https://whimsy.apache.org/roster/ppmc/)
+1. If they have already filed an ICLA, request creation of the committer account.
+   If they need to change anything in a previously filed ICLA, wait until the new ICLA is filed,
+   then request the account.
+   1. Wait until root says it is done
+   1. PMC Chair updates LDAP group membership which enables svn, gitbox and other access.
+      If the committer uses GitHub, they are responsible for linking it to their ASF account.
+   1. Add committer to the appropriate groups in Jira and CWiki
+1. Notify the committer of completion following the template below
+1. If committer is also to be a PMC member, PMC Chair or another PMC member sends email to board@ notifying
```

Comment:
Is this still valid? We had this discussion recently about notifications being not needed where @sebbASF wanted to be sure that the automated notification is happening - and presumably there should be no need to notify `board@` any more. Or maybe the case where "committership" and "pmc membership" is an exception and notification stil should be made. @sebbASF - would. your automation cover this case?

Reply:
Nope. You're right. This has changed. I'll get the current story,.